### ISSUE TITLE
Update installation instructions

### ISSUE THREAD

**Issue by pietercolpaert** [tdt/docs#32]
* [ ] Installation instructions should be updated to apache 2.4.
 * [ ] pagodabox should be removed as since an update at pagoda, it doesn't work anymore

**Comment by brechtvdv** [comment]
php5-mcrypt has to be installed and enabled too

**Comment by coreation** [comment]
Yes, good call. Although in next versions of Laravel this won't be necessary anymore :).

**Comment by Dobbelsteen** [comment]
Should I add 'php5-mcrypt' to the docs then? Seems like you will be using Laravel 4.x for some time still

**Comment by coreation** [comment]
@pietercolpaert what instructions should there be added to comply with Apache 2.4?
